src/sa_web/static/js/views/place-list-view.js

```diff
@@ -91,9 +91,11 @@ var Shareabouts = Shareabouts || {};
       $itemViewContainer.empty();
 
       this.collection.each(function(model) {
-        $itemViewContainer.append(self.views[model.cid].$el);
-        // Delegate the events so that the subviews still work
-        self.views[model.cid].supportView.delegateEvents();
+        if (self.views[model.cid]) {
```

Comment:
That's odd - it seems like `self.views` is always undefined. And it looks like `onAfterItemAdded` (on line 81 above) is never called either.